### ISSUE TITLE
Announcement of sunsetting doctrine/cache

### ISSUE THREAD

**Issue by pamtbaau** [getgrav/grav#3234]
I presume this will be know to the team, but in case this is not...: [Sunsetting doctrin/cache](https://github.com/doctrine/cache/issues/354)

**Comment by mahagr** [comment]
Yeah. We have already PSR-16 wrapper, which can be used instead of relying upon Doctrine Cache. I guess that we have to update all our code to use it and give instructions for the other devs to do the same.

**Comment by mahagr** [comment]
I mean plugin developers as they would need to change their caching calls.

**Comment by pamtbaau** [comment]
I presumed I had to do some tweaking of my own plugin. But good to know for certain.

When will you be giving instructions for other devs?
